airbyte-cdk/java/airbyte-cdk/db-sources/src/testFixtures/kotlin/io/airbyte/cdk/integrations/debezium/CdcSourceTest.kt

```diff
@@ -781,7 +781,10 @@ abstract class CdcSourceTest<S : Source, T : TestDatabase<*, T, *>> {
                 modelsSchema(),
             )
         } else {
-            assertExpectedStateMessageCountMatches(stateMessages1, MODEL_RECORDS.size.toLong())
+            assertExpectedStateMessageCountMatches(
```

Comment:
nit : Can you add a quick comment as to why (since MODEL_RECORDS_2) is sending state now